src/sagemaker_sklearn_container/handler_service.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Should we update the year to 2020?

Reply:
Looks like all files with this copyright note say a mix of 2018 and 2019. Let's update all of them to be consistent.

Reply:
Changed all the touched files to 2019-2020